runtime/gc_base/modronapi.cpp

```diff
@@ -1150,6 +1150,16 @@ j9gc_reinitialize_for_restore(J9VMThread *vmThread, const char **nlsMsgFormat)
 		goto _error;
 	}
 
+#if defined(OMR_GC_MODRON_SCAVENGER)
+	if (extensions->scavengerEnabled && (NULL != extensions->scavenger)) {
+		if (!extensions->scavenger->reinitializeForRestore(env)) {
```

Comment:
I'd rather call this from ParallelGlobalGC::reinitializeForRestore
Admittedly, it would harder to report Scavenger specific error...

Reply:
Removed, will need to be done on OMR side now.

Reply:
>Admittedly, it would harder to report Scavenger specific error...


> Scavenger reinit will only fail if alloc of sublists fails, which will be due to OOM.

This is the case for all of Parallel Global GC reinits, I don't think there's much value in being specific.